### PR TITLE
Stabilization: RHEL 8 STIG V1R11

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
This PR updates the RHEL 8 STIG to V1R11.

#### Rationale:

* Keep the STIG updated
* Downstream processes

### CODE REVIEW

**Comment by vojtapolasek** [comment]
@Mab879  I think you accidentally put the SCAP file instead of XCCDF manual.

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/software/integrity/crypto/harden_sshd_macs_opensshserver_conf_crypto_policy/tests/rhel8_stig_incorrect_policy.fail.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 # platform = Oracle Linux 8,Red Hat Enterprise Linux 8
-# profiles = xccdf_org.ssgproject.content_profile_stig
+# variables = sshd_approved_macs=hmac-sha2-512,hmac-sha2-256,hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com
+=stig_extended
```
[the remark this answers is not in the document]
Comment (Mab879) [reply linux_os/guide/system/software/integrity/crypto/harden_sshd_macs_opensshserver_conf_crypto_policy/tests/rhel8_stig_incorrect_policy.fail.sh]: Fixed.

**Review by marcusburghardt — APPROVED** [review]
LGTM. Thanks

**Comment by vojtapolasek** [comment]
I can confirm that fails of Automatus on Fedora are acceptable... this rule is not designed for Fedora and I tested it is working on RHEL 8.

**Pull request merged**